### PR TITLE
[PKG-2502] minimal-snowplow-tracker 0.0.2

### PR DESCRIPTION
**Pull request by skupr-anaconda**
Requirements: the file `setup.py` is inside source distribution on PyPi https://files.pythonhosted.org/packages/e4/9f/004f810169a48ed5c520279d98327e7793b6491f09d42cb2c5636c994f34/minimal-snowplow-tracker-0.0.2.tar.gz 

Actions:
1. Clean up the recipe:
- Remove `noarch` python
- Add flags `--no-deps --no-build-isolation` to `script`
- Add missing `setuptools` and `wheel` to `host`
2. Update `home` url
3. Add `doc_url` and a comment about missing `dev_url`
4. Add `description`
5. Skip a lint `missing_dev_url`

Notes:
- `dbt` requires it https://github.com/dbt-labs/dbt-core/blob/v1.5.3/core/setup.py#L58

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,45 +1,51 @@
...
-    - pip check
   requires:
     - pip
+  commands:
+    - pip check
 
 about:
-  home: https://www.fishtownanalytics.com
-  dev_url: https://pypi.org/project/minimal-snowplow-tracker/
+  home: https://www.getdbt.com/
```
Comment (rafaelmartins) [inline recipe/meta.yaml:38]:
```suggestion
  home: https://www.getdbt.com/
  dev_url: https://github.com/dbt-labs/snowplow-python-tracker
```

**Review by rafaelmartins — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml:38]: No, it's another package with other version numbers. There is no v0.0.2

Comment (rafaelmartins) [reply recipe/meta.yaml:38]: The files you are installing are from that repository, yes: https://github.com/dbt-labs/snowplow-python-tracker/commit/bf630807ab93326ada84044a06a77ce2e9ce7b33

you can compare the date of this commit also with the date of the upload in pypi. also compare the file content, last commit data, readme in pypi, etc.

the snowplow-tracker package with newer versions is avaliable somewhere else. the tags are just poorly maintained. you can ignore them

Comment (rafaelmartins) [reply recipe/meta.yaml:38]: the new, full, "non-minimal" snowplow-tracker is here: https://github.com/snowplow/snowplow-python-tracker. please note the different organizations :)

Comment (skupr-anaconda) [reply recipe/meta.yaml:38]: But there isn't a tag for v0.0.2. `dbt` requires **==0.0.2** https://github.com/dbt-labs/dbt-core/blob/v1.5.3/core/setup.py#L58

Comment (rafaelmartins) [reply recipe/meta.yaml:38]: a git tag is just a convention, nothing in pip or conda, etc, relies on tags. as long as the maintainer uploaded the source to pypi it works, no matter if tags are correct or not

Comment (skupr-anaconda) [reply recipe/meta.yaml:38]: Fixed

**Review by SC426 — APPROVED** [review]
LGTM with some suggested (but not necessary) changes:

Summary: Snowplow event tracker for Python.

Description: With this tracker you can collect event data from your Python-based applications, games or Python web servers/frameworks.

**Comment by skupr-anaconda** [comment]
> With this tracker you can collect event data from your Python-based applications, games or Python web servers/frameworks

Thanks. Updated!

**Pull request merged**